### PR TITLE
Open model hub doc

### PR DESCRIPTION
**Pull request by vincenthoogsteder**
Docs page to send users of the open-mobel-hub demo notebook to at the end of that.

Tried to align with the current state + what will be coming.

URL to notebook to be updated once we have final link to jupterlite

### CODE REVIEW

**Comment by vincenthoogsteder** [comment]
@ivarpruijn fyi

File: docs/docs/open-model-hub.md [hunk 1]
```diff
@@ -6,22 +6,24 @@ title: Open Model Hub
...
+All models are compatible with datasets that have been validated against the [open analytics taxonomy](/taxonomy/introduction.md). The source is available for all models and you’re free to make any changes. You can use the included pandas-compatible [Bach modeling library](/modeling/intro.mdx) to customize them, or even add in advanced ML models.
 
-Each model can run on an [Objectiv Bach DataFrame](/modeling/dataframe/bach.DataFrame.mdx) and delivers output in this format as well, so it can be used for further modeling.
+Input and output of the models are in the [Objectiv Bach DataFrame](/modeling/dataframe/bach.DataFrame.mdx) format, which enables you to chain multiple models together.
 
-:::info WIP
-We've setup an intial showcase of the available product analytics models in our [live demo notebook](https://notebook.objectiv.io/lab/?path=product_analytics.ipynb). Right now, we are creating functions for each of these models, so you can easily use then from a notebook.
+:::demo notebook
+We've setup an intial showcase of the available product analytics models in our [open model hub demo notebook](https://notebook.objectiv.io/).
```
Comment (ivarpruijn) [inline docs/docs/open-model-hub.md:16]:
```suggestion
We've set up an initial showcase of the available product analytics models in our [open model hub demo notebook](https://notebook.objectiv.io/). 
```

File: docs/docs/open-model-hub.md [hunk 2]
```diff
@@ -32,22 +34,22 @@ Each model can run before or after any of the available other models. This allow
...
 * `session_duration`: Mean session duration.
 * `duration_between_events`: Mean session duration between specific defined events.
 * `bounce_rate`: Sessions with only a single hit.
 
+## Available filters
+Filters can be applied to each of the models. Initially, there is a first filter to quickly zoom into new users or sessions.
+* `is_first_session`: The first-ever session of a user.
+
 ## Models in development
 * `user clusering`: Grouping uses based on similar behaviour in using product features.
```
Comment (ivarpruijn) [inline docs/docs/open-model-hub.md:48]:
```suggestion
* `user clustering`: Grouping uses based on similar behavior in using product features.
```

**Review by ivarpruijn — COMMENTED** [review]
Nice, suggestions for some typos

File: docs/docs/quickstart-guide.md [hunk 3]
```diff
@@ -54,22 +52,24 @@ To generate event data yourself, simply go to the local version of the objectiv
...
 http://localhost:8080/
 ```
-
 When you trigger an event, you can see a request show up in your docker logs. If you used a terminal to spin up the containers, it shows there as well. 
 
-:::info
-Please note that all events have a UTC timestamp. To see/use your new events in the Demo Notebook, just rerun the 'Explore the data' cell.
-:::
+### Accessing Metabase
+To view the data that has been put out to Metabase, open the following URL in your browser:
```
Comment (borft) [inline docs/docs/quickstart-guide.md:58]: should be include credentials here? they are also printed in the notebook

Comment (vincenthoogsteder) [reply docs/docs/quickstart-guide.md:58]: great point, let's add them, doing now

File: docs/docs/introduction.md [hunk 4]
```diff
@@ -12,21 +12,16 @@ import Mermaid from '@theme/Mermaid';
...
+With Objectiv, you can take, stack and run [pre-built data models](/open-model-hub) off the shelf to quickly build highly specific model stacks for in-depth product analytics and exploration. Objectiv's reusable modeling is enabled by validating all collected data against an [open analytics taxonomy](/taxonomy) to ensure it is clean, consistent and structured with effective modeling in mind.
 
-<img src={useBaseUrl('/img/objectiv-pipeline.svg')} alt="Objectiv Pipeline" class="img-l" />
+<img src={useBaseUrl('/img/objectiv-overview.svg')} alt="Objectiv Overview" class="img-l" />
 
-With [Objectiv Bach](https://www.objectiv.io/docs/modeling/) (our data modeling library), you can use familiar Pandas-like DataFrame operations on your full SQL dataset. It includes extensions for effective feature creation for datasets that embrace the [taxonomy](/taxonomy).
+With the included [Objectiv Bach](/modeling) modeling library, you can customize or build models with familiar Pandas-like dataframe operations that run on your full SQL dataset. When you're ready to share your findings, you can output to open-source BI tools with a single command.
```
[the remark this answers is not in the document]
Comment (vincenthoogsteder) [reply docs/docs/introduction.md:21]: as discussed great point and let's put your brain on this

File: docs/docs/open-model-hub.md [hunk 5]
```diff
@@ -32,18 +30,18 @@ Each model can run before or after any of the available other models. This allow
 * `frequency`: Users categorized by number of total sessions.
```
[the remark this answers is not in the document]
Comment (vincenthoogsteder) [reply docs/docs/open-model-hub.md:30]: good point, updating

**Pull request merged**